app/models/manageiq/providers/openstack/infra_manager/refresher.rb

```diff
@@ -12,6 +12,35 @@ def save_inventory(ems, target, hashes)
         EmsRefresh.queue_refresh(ems.network_manager)
       end
 
+      def collect_inventory_for_targets(ems, targets)
+        return [[ems, nil]]
+      end
+
+      def parse_targeted_inventory(ems, _target, inventory)
+        log_header = format_ems_for_logging(ems)
+        _log.debug("#{log_header} Parsing inventory...")
+        hashes = parse_legacy_inventory(ems)
+        _log.debug("#{log_header} Parsing inventory...Complete")
+        hashes
+      end
+
+      def preprocess_targets_manager_refresh
+        @targets_by_ems_id.each do |ems_id, targets|
+          ems = @ems_by_ems_id[ems_id]
+          next unless ems.inventory_object_refresh?
```

Comment:
@d-m-u since this doesn't support inventory_object_refresh think this method should just be empty